scalableminds/webknossos
# in task-from-file creation, allow multiple files in formData

### URL of deployed dev instance (used for testing):
- https://taskmultifiles.webknossos.xyz

### Steps to test:
- in create task form, select »Upload NML file«, drag in multiple files (you may mix nmls and zips)
- single file should also still work

### Issues:
- fixes #3175 

------
